addons/website/static/src/js/website.contentMenu.js

```diff
@@ -236,6 +238,7 @@ var EditMenuDialog = widget.Dialog.extend({
     add_menu: function () {
         var self = this;
         var dialog = new MenuEntryDialog(this, {menu_link_options: true}, undefined, {});
+        dialog.$modal.addClass('o_contentMenu_link_modal');
```

Comment:
My question was general, I did not check the LESS, in most cases you should not have to style elements other than the modal-body content ;) but if this is the case here, ok ;)